pkg/kapp/config/default.go

```diff
@@ -72,21 +72,39 @@ rebaseRules:
       apiVersion: v1
       kind: ServiceAccount
 
-- path: [spec, storageClassName]
+# PVC
+- path: [metadata, annotations, pv.kubernetes.io/bind-completed]
   type: copy
-  sources: [new, existing]
-  resourceMatchers:
+  sources: [existing]
+  resourceMatchers: &pvcs
   - apiVersionKindMatcher:
       apiVersion: v1
       kind: PersistentVolumeClaim
 
+- path: [metadata, annotations, pv.kubernetes.io/bound-by-controller]
+  type: copy
+  sources: [existing]
+  resourceMatchers: *pvcs
+
+- path: [metadata, annotations, volume.beta.kubernetes.io/storage-provisioner]
+  type: copy
+  sources: [existing]
```

Comment:
i think it's still worth keeping `[new, existing]` for this one and above. wdyt?

Reply:
This is actually copied from https://github.com/k14s/kapp/blob/master/docs/rebase-pvc.md Shall we update that as well?

Reply:
probably.

Reply:
Done for both